hever/sass/_config-child-theme-deep.scss

```diff
@@ -18,25 +18,25 @@ $typescale-ratio: 1.15; // Run ratio math on 1em == $typescale-base * $typescale
 
 $config-global: (
 
-	/* Fonts */
+	
```

Comment:
We shouldn't remove the comments from sass files

Reply:
This seems to be side effect of lint fixes. reverted the change